packages/blockchain-extension/extension/commands/deleteEnvironmentCommand.ts

```diff
@@ -31,13 +32,13 @@ export async function deleteEnvironment(environment: FabricEnvironmentTreeItem |
             // If called from command palette
             // Ask for environment to delete
             // First check there is at least one that isn't local_fabric
```

Comment:
can we change the comment to reflect the code

Reply:
Updated.